client/securedrop_client/gui/conversation/export/export_wizard_page.py

```diff
@@ -402,6 +402,10 @@ def update_content(self, status: ExportStatus, should_show_hint: bool = False) -
             header = _("Export sucessful, but drive was not locked")
             body = STATUS_MESSAGES.get(status)
 
+        elif status in self.UNRECOVERABLE_ERRORS:
+            header = _("Export Failed")
+            body = STATUS_MESSAGES.get(status)
+
```

Comment:
This will address the "Hanging" at "Working...." issue that cfm saw during ERROR_EXPORT. (Addresses the following very specific case: Unlocked, attached device -> launch export wizard -> ERROR_EXPORT)